Code_Exercises/Exercise_06_Vector_Add/source.cpp

```diff
@@ -6,6 +6,36 @@
 
  You should have received a copy of the license along with this
  work.  If not, see <http://creativecommons.org/licenses/by-sa/4.0/>.
+
+ * SYCL Quick Reference
+ * ~~~~~~~~~~~~~~~~~~~~
+ *
+ * // Default construct a queue
+ * auto q = sycl::queue{};
+ *
+ * // Declare a buffer pointing to ptr
+ * auto buf = sycl::buffer{ptr, sycl::range{n}};
+ *
+ * // Submit work to the queue
+ * q.submit([&](sycl::handler &cgh) {
+ *   // COMMAND GROUP
+ * });
+ *
+ * // Within the command group you can
+ * //    1. Declare an accessor to a buffer
+ *          auto acc = sycl::accessor{buf, cgh};
```

Comment:
Maybe we could add different variable names for these, just in case people copy-paste it, would also allow us to highlight here that the default is read_write?

Reply:
Good idea. Will update